### PR TITLE
BZ1924568 Added known issue to 4.8 RN

### PR DESCRIPTION
**Pull request by ctauchen**
* applies only to `enterprise-4.8`
* see [BZ#1924568](https://bugzilla.redhat.com/show_bug.cgi?id=1924568)
* needs QE @xiuwang 
* [Direct Preview Link](https://deploy-preview-34362--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-8-release-notes?utm_source=github&utm_campaign=bot_dp#ocp-4-8-known-issues)

### CODE REVIEW

File: release_notes/ocp-4-8-release-notes.adoc [hunk 1]
```diff
@@ -2311,6 +2311,8 @@ For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=18786
 
 * To assist with troubleshooting, logs collected on boostratp failures by the installer now include IP addresses and routes for the control plane and bootstrap hosts. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1956079)[*BZ#1956079*])
 
+* When using a self-signed C2S Amazon Commercial Cloud Services cluster, you cannot pull from or push to an internal image registry. As a workaround, you must set `spec.disableRedirect` to `true` in the `configs.imageregistry/cluster` resource. This lets you pull the image layers from the image registry rather than directly from S3 storage. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1924568[*BZ#1924568*])
```
Comment (xiuwang) [inline release_notes/ocp-4-8-release-notes.adoc:2314]: s/C2S Amazon Commercial Cloud Services cluster/Amazon Commercial Cloud Services cluster(C2S)

Comment (ctauchen) [reply release_notes/ocp-4-8-release-notes.adoc]: Yes, of course. Changed, but removed the acronym, seeing as we have no second reference.

Comment (xiuwang) [reply release_notes/ocp-4-8-release-notes.adoc]: You are professional, thanks

**Comment by xiuwang** [comment]
LGTM

**Comment by mburke5678** [comment]
@ctauchen LGTM

**Review by bobfuru — APPROVED** [review]

**Pull request merged**